test/command_line_test.rb

```diff
@@ -90,61 +89,63 @@ def test_volume
   end
 
   def test_track
-    flexmock(@player).should_receive(:track).once.and_return("Foo")
-    flexmock($stdout).should_receive(:puts).once.with("Foo")
+    flexmock(@player).should_receive(:track).once.and_return('Foo')
+    flexmock($stdout).should_receive(:puts).once.with('Foo')
     Anyplayer::CommandLine.parse(%w(track))
   end
 
   def test_artist
-    flexmock(@player).should_receive(:artist).once.and_return("Foo")
-    flexmock($stdout).should_receive(:puts).once.with("Foo")
+    flexmock(@player).should_receive(:artist).once.and_return('Foo')
+    flexmock($stdout).should_receive(:puts).once.with('Foo')
     Anyplayer::CommandLine.parse(%w(artist))
   end
 
   def test_album
-    flexmock(@player).should_receive(:album).once.and_return("Foo")
-    flexmock($stdout).should_receive(:puts).once.with("Foo")
+    flexmock(@player).should_receive(:album).once.and_return('Foo')
+    flexmock($stdout).should_receive(:puts).once.with('Foo')
     Anyplayer::CommandLine.parse(%w(album))
   end
 
   def test_name
-    flexmock(@player).should_receive(:name).once.and_return("Foo")
-    flexmock($stdout).should_receive(:puts).once.with("Foo")
+    flexmock(@player).should_receive(:name).once.and_return('Foo')
+    flexmock($stdout).should_receive(:puts).once.with('Foo')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.